lib/bootstrap.js

```diff
@@ -4,11 +4,23 @@ const cmdArgs = require('command-line-args');
 
 require('./utils/process-log');
 
+const helpFlagPresent = args => args.find(arg => ['help', '--help'].indexOf(arg) !== -1);
+
+const normalizeFlags = (args, command) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== command.name && arg !== command.alias);
+
+const isCommandUsed = commands => commands.find(cmd => (cmd.alias ? process.argv.includes(cmd.name) || process.argv.includes(cmd.alias) : process.argv.includes(cmd.name)));
```

Comment:
The previous version was more readable. Would you mind keep that?

Reply:
```js
arg !== command.name && arg !== command.alias
```
The respective statement appear to be redundant atleast IMO.

Reply:
```js
process.argv.includes(cmd.name) || process.argv.includes(cmd.alias)
```
same here :thinking: